homeassistant/components/keyboard_remote/__init__.py

```diff
@@ -295,6 +306,7 @@ async def async_monitor_input(self, dev):
                 _LOGGER.debug("Start device monitoring")
                 await self.hass.async_add_executor_job(dev.grab)
                 async for event in dev.async_read_loop():
+                    # pylint: disable=no-member
```

Comment:
Why is there a pylint error now?

Reply:
I'm not sure.

pylint complains that `ecodes` has no member `EV_KEY`, when it does, and works fine when executed.

Reply:
Upon further investigation, it appears that `ecodes` uses a C extension that pylint may not support verifying.